Gemfile

```diff
@@ -28,6 +28,8 @@ group :development, :test do
   gem 'capybara'
   gem 'byebug'
   gem 'pry'
+  gem 'brakeman', require: false
```

Comment:
brakeman is already used by Code Climate - are you using CC in this project? http://docs.codeclimate.com/article/160-glossary-oss-tools-used-by-code-climate